src/App.tsx

```diff
@@ -1,26 +1,239 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useRef, useState } from 'react';
 import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import { NewTodo } from './components/NewTodo/NewTodo';
+import { ToggleAllButton } from './components/ToggleAllButton/ToggleAllButton';
+import { TodoList } from './components/TodoList/TodoList';
+import { Todo } from './types/Todo';
+import { TodoFilter } from './components/TodoFilter/TodoFilter';
+import { ErrorNotification } from './components/ErrorNotification';
+import * as todoService from './api/todos';
+import { FiltersEnum } from './types/enums/FiltersEnum';
+// comment to restart pipeline
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [currentFilter, setCurrentFilter] = useState<FiltersEnum>(
+    FiltersEnum.All,
+  );
+  const [todoTitle, setTodoTitle] = useState<Todo['title']>('');
```

Comment:
it will be a string anyways
```suggestion
  const [todoTitle, setTodoTitle] = useState('');
```